### PR TITLE
test: Make sure the status of a compose is called status

### PR DESCRIPTION
**Pull request by Gundersanne**
This was a bit confusing because in the inline documentation I wrote
used state as a key. However this changed to status pretty early on and
I forgot to update it.

### CODE REVIEW

File: src/test/SmartComponents/redux/reducers.test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import types from '../../../SmartComponents/redux/types';
 
 const compose = {
     'xxxx-xxxx-xxxx-xxxx': {
-        state: 'uploading',
+        status: 'uploading',
```
Comment (atodorov) [inline src/test/SmartComponents/redux/reducers.test.js:6]: See also `redux/actions.test.js`

**Review by atodorov — CHANGES_REQUESTED** [review]
Overall looks good, 1 more file needs patching.

File: src/test/SmartComponents/redux/reducers.test.js (hunk 1 above)
Comment (Gundersanne) [reply src/test/SmartComponents/redux/reducers.test.js:6]: Nice,  I also exchanged the key with a dummy uuid.

File: .eslintrc.yml [hunk 2]
```diff
@@ -49,7 +49,7 @@ rules:
     - error
     - 4
     - SwitchCase: 1
-      MemberExpression: 0
+      MemberExpression: 1
```
Comment (Gundersanne) [inline .eslintrc.yml:52]: Multi-line statements would get indented on the same level with MemberExpression 0, I think have an indent makes it more readable.

Comment (atodorov) [reply .eslintrc.yml:52]: I was bashing my head against the wall yesterday with exactly the same problem. Thanks for figuring it out.

**Review by atodorov — APPROVED** [review]

**Pull request merged**